### ISSUE TITLE
Using with browserify and Babeljs fails when bundling

### ISSUE THREAD

**Issue by jbruggem** [fmoo/react-typeahead#75]
Due to babeljs being able to convert JSX to JS, a situation can arise where reactify is not a dependency of the project.

In that case, bundling with browserify will fail due to reactify not being present.

This is due to the entry in `package.json`:
```js
  "browserify": {                                                    
    "transform": [                                                   
      "reactify"                                                     
    ]                                                                
  },   
```

**Comment by fmoo** [comment]
Browserify is only listed in devDependencies.

When bundling from a release, you should get the precompiled sources via main and not need babel or browserify.

Maybe I'm missing something?  What command are you running and do you have repro steps?

**Comment by jbruggem** [comment]
Thanks for your quick reply!

Here's my context:
- I use npm to handle dependencies (including this one)
- I use a gulp task to transform my code to ES5
- I use another gulp task to bundle everything for browser consumption. Dependencies are in one file (libs), my code in another (main). 

When compiling 'libs', browserify complains about not being able to load 'reactify'. Since the only reference in react-typeahead to reactify is the aforementioned bit of code, I am assuming browserify tries to apply the 'transform' when bundling. Since 'reactify' is not in my dependencies, it fails.

I hope this makes my issue clearer. If not, I'll try to build a minimal reproducible bit of code as soon as possible.

**Comment by jbruggem** [comment]
Sorry; the issue is solved on the repo, since the 'browserify' entry has been removed in package.json (it is not solved in the latest version on NPM).

**Comment by fmoo** [comment]
@jbruggem - Ahhh, that explains it!  I just published a 1.0.5 to npm, so feel free to use the latest release instead of the development version.
